README.md

```diff
@@ -0,0 +1,168 @@
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+ Snapscan
+ 
+ - [📗 Table of Contents](#-table-of-contents)
+- [📖 Snapscan ](#-snapscan-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+    
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Snapscan <a name="about-project"></a>
+
+
+A mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+
+<div align="center">
+  <img src="./app/assets/images/diagram.png" alt="logo" />
+  <br/>
+</div>
+
+
+## 🛠 Built With <a name="built-with"></a>
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server Side</summary>
+  <ul>
+    <li><a href="https://www.rubyonrails.org/en/">Ruby on Rails</a></li>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://www.w3.org/Style/CSS/Overview.en.html">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+ 
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+- **Splash Screen**
+- **Sign up and log in pages**
+- **Home page (categories page)**
+- **Transactions page**
+- **Add a new category page**
+- **Add a new transaction page**
+- **Use devise for user authentication**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+- <a href="https://snapscan-89o4.onrender.com/">Live Demo</a>
+- [Video Presentation](https://www.loom.com/share/a5387e7ca63342988e5defaa5607dd94?sid=55bafea5-ea2f-4bec-ac24-3ee12e540bb7)
+
```

Comment:
- [ ] To meet the project requirements, I suggest that you include a highlight in your video presentation that showcases a piece of code you find interesting or are proud of creating. Therefore, ensure that you display some code in the presentation and explain what makes you feel proud. :pray:

Reply:
Hello @Lizdev-05, I mentioned in the video about the piece of code(method) I added to the group model to limit it to just 5 transactions. I also mentioned the transitions I added to the css styling. Both of these are not part of the requirements but I added and mentioned them in the video.
Kindly attend to this as soon as you can, thank you.